utilities/emoji.json

```diff
@@ -0,0 +1,23 @@
+{
+    "happy": [
+        ":smile:",
+        ":partyparrot:",
+        ":logan_cheesin:",
+        ":dan:",

```

Comment:
I'm not sure this is "happy," and I'd prefer it to go away altogether

Reply:
You're happy and :dan: will last as long as :logan_cheesin: but it's simple JSON and can be removed

Reply:
You're happy and :dan: will last as long as :logan_cheesin: but it's simple JSON and can be removed